Add files via upload

kuurasuo opened the pull request.

citlali-trigos-raczkowski commented on tweet_processing.py:
```diff
@@ -0,0 +1,71 @@
+data = []
+cols = ["tweets"]
+
+df1 = pd.read_csv("sarw.csv")
+df2 = pd.read_csv("info.csv")
+df3 = pd.read_csv("uscg.csv")
+df4 = pd.read_csv("last_week.csv")
+
```
I can't run this without these CSVs

Please (a) add them in if they're going to be used in the project, or (b) just the concatenated one on line 16

citlali-trigos-raczkowski commented on Twitter.py:
```diff
@@ -0,0 +1,62 @@
+necessitated the Elevated Access to the API, and our request was approved. The twitter "search" function of the API was able to give \
+results for the last 7 days. A thousand tweets were downloaded from the users @USCGSoutheast, @SARwatchMED and @InfoMigrants. \
+Since we are focusing on deaths and disappearances, I scrapped tweets that did not contain any of the words \
+"migrant/s", "dead", "death", "died", "bodies", or "missing". I attempted to create a model to extract workable data like "20 missing" etc. \
```
We're using "we" pronouns throughout, can you update the "I"s to match that?

citlali-trigos-raczkowski commented on Twitter.py:
```diff
@@ -0,0 +1,62 @@
+from the tweets by looking at the words that are in the vincinity of relevant keywords. \
+This however proved to be futile, as the reports have different as the full picture could really not be gained this way. Relevant context \
+was missing and as there were many sources and \
+not all contained specific enough data, it is difficult to distinguish wheter or not two tweets were talking about the same incident. The list contained only \
```
whether

citlali-trigos-raczkowski commented on Twitter.py:
```diff
@@ -0,0 +1,62 @@
+not all contained specific enough data, it is difficult to distinguish wheter or not two tweets were talking about the same incident. The list contained only \
+95 tweets, so getting the data manually was a far more preferable approach. We were able to extract the location of incident, number \
+dead/missing, and date of the \
+report. After doing this, I inferred the route from the tweets, and added that as well. Nearly all of the data was from the mediterranian, \
```
MediterranEan

citlali-trigos-raczkowski commented on Twitter.py:
```diff
@@ -0,0 +1,62 @@
+dead/missing, and date of the \
+report. After doing this, I inferred the route from the tweets, and added that as well. Nearly all of the data was from the mediterranian, \
+which is not surpring, since \
+according to MMP data mediterranian routes are the ones with most incidents and also due to the fact that one the users war @SARwatchMED, which \
```
MediterranEan

citlali-trigos-raczkowski commented on Tweet_Processing.py:
```diff
@@ -0,0 +1,165 @@
+print(dm[0])
+print(dates[0])
+
+#for tweet in dm:
+#    l = tweet.split()
+#    for i in range(len(l)):
+#        s = l[i].strip(',.:;')
+#        st = ''
+#        if s == 'dead' or s == 'died' or s == 'death' or s == 'bodies':
+#            st += 'dead : ' + str(l[i-3]) + ' ' + str(l[i-2]) + ' ' + str(l[i-1]) + ' ' + str(l[i])
+#        elif s == 'missing':
+##            st += 'missing : ' + str(l[i-3]) + ' ' + str(l[i-2]) + ' ' + str(l[i-1]) + ' ' + str(l[i])
+#        if st != '':
+#            strs.append(st)
```
Could you remove any comments / clean up the code please

citlali-trigos-raczkowski committed:
feat(twitter): Jani's part

citlali-trigos-raczkowski commented:
Could you add more detail to how you got the data from the tweets? It's not obvious how you got the plot data from the introductory text in the twitter page

citlali-trigos-raczkowski commented:
Also maybe the side panel menu item should be renamed to something more informative than just "Twitter"

Something more relevant to the project

kuurasuo committed:
Update Twitter.py

kuurasuo committed:
Update Tweet_Processing.py

kuurasuo committed:
Rename Twitter.py to Recent_data_from_Twitter.py

kuurasuo commented:
> Also maybe the side panel menu item should be renamed to something more informative than just "Twitter"
> 
> Something more relevant to the project

Would "Recent data from Twitter" be okay? I think it would highlight that the reason we wanted to enrich the dataset was to get more current data.

citlali-trigos-raczkowski commented:
I think _Recent data from Twitter_ sounds good. @srodriguezb ?

Seeing a couple things: 
1. Use of "mediterranian" throughout instead of _Mediterran*e*an_ (Capitalized with an e) (including in the data)
2. Typo in the filter menu:
<img width="278" alt="Screen Shot 2022-10-23 at 19 37 11" src="https://user-images.githubusercontent.com/53492091/197404304-166a32e9-7805-4557-b522-e15aee1d87a2.png">

As an idea to bulk up this page, do you want to maybe show the dataframe table? 
```
st.dataframe(df)
```
Maybe show a couple tweets? You can also link the listed twitter handles to the actual twitter page. 
```
st.markdown("attach a link like this: [link name](www.link.com) ")
```

srodriguezb commented:
Yeah, the name is good. Also, the typo comes from the data right ? I will check it and fix it.

kuurasuo committed:
Add files via upload

The new data.csv fixes the spelling error, and there is now an embedded Tweet, links, and part of the dataframe. Thank you Lali for the suggestions, the page looks much better now!

kuurasuo committed:
Add files via upload

Cleaned the code further in Tweet_processing.py. Added coordinates of the approximate locations of the incidents from the dataset to data.csv, and using these beefed up the Twitter page with a similar map of incident locations as in the Home page.
